citi-onboarding/Recorda-one-page
# Feature/feedback integration

### **What I did:**
- made the integration for the feedback section

### **How to test:**
- git checkout main
- git pull
- git checkout feature/feedback_integration
- Add some content to the news database and check if it is being shown on frontend
